packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -105,12 +119,13 @@ export interface ModifyDetach<TNodeChange = NodeChangeType> extends HasOpId {
     changes: TNodeChange;
 }
 
-export interface Reattach extends HasOpId {
+// TODO: Reattach and ModifyReattach shouldn't have tiebreak field
```

Comment:
Feel free to split the tiebreak off from the HasPlaceFields or however you'd want it refactored.